src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1852,6 +1852,21 @@
   <data name="UsesPreviewTypeParameterMessage" xml:space="preserve">
     <value>'{0}' uses the preview type '{1}' and needs to opt into preview features. See {2} for more information.</value>
   </data>
+  <data name="SpecifyCultureForToLowerAndToUpperTitle" xml:space="preserve">
+    <value>Specify culture explicitly</value>
```

Comment:
```suggestion
    <value>Use invariant version or specify culture explicitly</value>
```

Reply:
@buyaa-n To me `Use invariant version or specify current culture` implies that there are only those two options when it's actually possible to specify any culture. Would `Specify a culture or use an invariant version` be more suitable?

Reply:
> Would `Specify a culture or use an invariant version` be more suitable?

Yes that sounds better, thanks